src/core/include/openvino/opsets/opset1_tbl.hpp

```diff
@@ -134,6 +134,7 @@ _OPENVINO_OP_REG(Sinh, ov::op::v0)
 _OPENVINO_OP_REG(Softmax, ov::op::v1)
 _OPENVINO_OP_REG(Sqrt, ov::op::v0)
 _OPENVINO_OP_REG(SpaceToDepth, ov::op::v0)
+_OPENVINO_OP_REG(SparseConv, ov::op::v1)
```

Comment:
Leads to an error:

```
/agent/_work/1/openvino/src/core/include/openvino/opsets/opset1_tbl.hpp:137:18: error: ‘ov::op::v1::SparseConv’ has not been declared
  137 | _OPENVINO_OP_REG(SparseConv, ov::op::v1)
```

Reply:
@jane-intel, may I ask what I've missed in PR that SparseConv is not added to `opset1`? I use oset1 for now just for validation